### PR TITLE
feat: update to fundamental-styles@0.4.0-rc.3

### PR DESCRIPTION
**Pull request by meganaconley**
### Description
[BREAKING CHANGE]

Updates fundamental-styles to 0.4.0-rc.3, bringing in the following markup changes:

1. `SideNav` refactor
- Now uses `nested-list` styles and structure
- Added `utility` nav
2. `Button` changes
- `medium` button removed
3. `Calendar` changes
- Month and year selectors changed from `ul` to `table`
- Adds role="button" to each selectable calendar item
- Adds aria-selected attribute to the currently-selected calendar item.

### CODE REVIEW

**Comment by greg-a-smith** [comment]
@jacobdevera No, it's not just you.  When the side nav collapses and it shown via the hamburger menu, it no longer scrolls.

File: src/Calendar/Calendar.js [hunk 1]
```diff
@@ -350,7 +391,7 @@ class Calendar extends Component {
         for (let index = 0; index < 7; index++) {
             weekDays.push(
                 <th className='fd-calendar__column-header' key={index}>
-                    <span className='fd-calendar__day-of-week'>
+                    <span className='fd-calendar__day-of-week' role='button'>
```
Comment (greg-a-smith) [inline src/Calendar/Calendar.js]: Why do the days of the week have a role of button?  They aren't actually clickable, are they?

File: src/SideNavigation/SideNav.js [hunk 2]
```diff
@@ -55,9 +57,10 @@ class SideNav extends Component {
 SideNav.propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    compact: PropTypes.bool,
+    condensed: PropTypes.bool,
```
Comment (greg-a-smith) [inline src/SideNavigation/SideNav.js]: Needs a prop description.

**Comment by skvale** [comment]
Related to the side-nav not scrolling from previous comments, the scrollbar changed elements

master branch

Screen Shot 2020-01-16 at 10 10 47 AM

this branch

Screen Shot 2020-01-16 at 10 09 29 AM

**Comment by greg-a-smith** [comment]
@skvale That's actually part of this PR's change.  The Side Navigation component (which is what the site's left nav is built with) has been refactored and is now using different classes.

**Comment by meganaconley** [comment]
New commit fixes:
- unnecessary button role on Calendar
- Button page layout
- SideNav scroll issue
- Prop descriptions

**Comment by greg-a-smith** [comment]
@meganaconley The PR feedback changes looked good except for the side nav scrolling.  It did scroll this time, however, now even the search box scrolled with it -- that is supposed to stay fixed.  Instead of more comments and screenshots, I contributed a few changes.  I set the sidebar width to `15rem` to match the side nav styles (to get rid of the gray gutter on the right of the side nav).  I moved the `overflow` definitions so only the side nav items would scroll.

**Comment by meganaconley** [comment]
Thanks for the help @greg-a-smith!

**Pull request merged**